plugins/source/k8s/client/spec/schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/cloudquery/cloudquery/plugins/source/k8s/client/spec/spec",
+  "$ref": "#/$defs/Spec",
+  "$defs": {
+    "Spec": {
+      "properties": {
+        "contexts": {
+          "oneOf": [
+            {
+              "items": {
+                "type": "string",
+                "minLength": 1
+              },
+              "type": "array"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "concurrency": {
+          "type": "integer",
+          "minimum": 1,
+          "default": 50000
+        }
+      },
+      "additionalProperties": false,
```

Comment:
Can we not enforce `additionalProperties: false`? Seems a bit too strict

Reply:
I think this means that if we add a new config property, update our docs to include, then someone copy pastes the docs and still uses an older version they'll get a validation error

Reply:
Currently it's not failing, only producing an err log because of https://github.com/cloudquery/plugin-sdk/pull/1280.

We can make a breaking change once we decide to actually fail on the wrong plugin cfg.